docs/eventing/broker/broker-types.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Broker types"
+weight: 01
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - docs/eventing/broker/alternate
+---
+
+The following broker types are available for use with Knative Eventing.
+
+## Multi-tenant channel-based broker
+
+Knative Eventing provides a multi-tenant (MT) channel-based broker implementation that uses channels for event routing.
+
+Before you can use the multi-tenant (MT) channel-based broker implementation, you must install a [channel provider](../channels/channel-types-defaults).
+<!--TODO: explain how channels are used for routing-->
+
+**NOTE:** InMemoryChannel channels are for development use only and must not be used in a production deployment.
+
+## Alternative broker implementations
+
+In the Knative Eventing ecosystem, alternate broker implementations are welcome as long as they
+respect the [broker conformance spec](https://github.com/knative/eventing/blob/main/docs/spec/broker.md).
+
+The following is a list of brokers provided by the community or vendors:
```

Comment:
Maybe we should add the class string used by each so someone can quickly look and see what is taken?

Reply:
Not sure what you mean, can you add it as a suggestion?